TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -506,12 +512,15 @@ public void DidChangeConfigurationParams(IEnumerable<string> arguments)
 
             var typeCobolOptions = new TypeCobolOptions(Configuration);
 
+            if (ClientConfigurationChangedEvent != null)
+                ClientConfigurationChangedEvent(this, arguments);
+
             //Configure CFG/DFA analyzer + external analyzers if any
             var analyzerProvider = new CompositeAnalyzerProvider();
-            analyzerProvider.AddActivator((o, t) => CfgDfaAnalyzerFactory.CreateCfgAnalyzer("cfg-dfa", Configuration.CfgBuildingMode));
+            this.AnalyzerProvider.AddActivator((o, t) => CfgDfaAnalyzerFactory.CreateCfgAnalyzer(TypeCobolLanguageServer.lspcfgId, CfgBuildingMode.Standard));
```

Comment:
There is a problem here, the `Configuration.CfgBuildingMode` is lost so this means that we currently do not support use of DFA features along with CFG graph view. Also the `UseCfgDfaRefresh` option is not used here but it should.

We need to think about the configuration and the way graphs are used. If we keep our current CFG/DFA implementation, we'll have to create 2 analyzers if the user want to visualize graphs and perform DFA analysis and this is decided here when we register activators.

Reply:
Also we need the `CompositeAnalyzerProvider` to add external quality analyzer.

Reply:
In LSP mode for the moment only CFG Standard mode is used for the CFGViewer, but you are right if DFA analysis is used we need a second analyzer. I remenber we were thinking a bout a solution to compute the DFA (Extended) Graph from a standard graph, by applying a transformation.